pages/Company-Info-Page.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
```

Comment:
- `Company-Info-Page.tsx` does not meet our naming convention rules. Read the guides in the `docs` repo about naming conventions and rename the file correctly.